environment.yml

```diff
@@ -48,7 +48,7 @@ dependencies:
   - mysql-connector-c
   - nanoflann <1.4
   - nlohmann_json=3.11.2
-  - ninja >=1.7.2,<1.8
```

Comment:
If we make any changes to the original environment.yml, it needs to be also updated in the recipe/meta.yaml.